projects/plugins/crm/src/automation/class-automation-bootstrap.php

```diff
@@ -140,6 +140,7 @@ protected function register_triggers(): void {
 	protected function register_conditions(): void {
 		$conditions = array(
 			\Automattic\Jetpack\CRM\Automation\Conditions\Contact_Field_Changed::class,
+			\Automattic\Jetpack\CRM\Automation\Conditions\Contact_Transitional_Status::class,
```

Comment:
This was a previously-created condition, but perhaps we should rename it to something simple, like `Contact_Status_Changed`. Or better yet, remove it, since I assume it's covered already in `Contact_Field_Changed`?